docs/extensions/tutorials/_posts/1970-01-02-WritingATheme.md

```diff
@@ -375,3 +375,52 @@ Check `Customize theme` under the `Style` tab. You can see the `Restaurants` sec
 <p class="image">
 <img src='{{ site.url }}/img/tutorials/settings-theme/style-tab-themes-customise_theme-color_picker_preview.png'/>
 </p>
+
+## Adding a Custom Font
+
+In order to add a custom font to a theme, you'll have to add a `fonts` folder to your extension's `app` folder and add your custom fonts to it in `.TTF` format. You can see this structure in our [open sourced]("https://github.com/shoutem/extensions/tree/master/shoutem-rubicon-theme/app/fonts") theme extensions.
```

Comment:
Can you prepare some free font so that people can use it as an example? Make sure that screenshots reflect the font you're using.